src/extension/background-script/router.ts

```diff
@@ -128,6 +128,8 @@ const routes = {
       getRelays: nostr.getRelays,
       encryptOrPrompt: nostr.encryptOrPrompt,
       decryptOrPrompt: nostr.decryptOrPrompt,
+      nip44EncryptOrPrompt: nostr.nip44EncryptOrPrompt,
```

Comment:
should we rename the existing methods to be prefixed with nip04?

Reply:
I can do that if you want, just let me know